src/renderers/dom/fiber/ReactDOMFiberEntry.js

```diff
@@ -813,6 +813,7 @@ const foundDevTools = injectInternals({
   // This is an enum because we may add more (e.g. profiler build)
   bundleType: __DEV__ ? 1 : 0,
   version: ReactVersion,
+  id: 'react-dom',
```

Comment:
`rendererName`? `rendererUniqueName`? People cloning ReactDOM might think a "type" would still be ReactDOM.